README.md

```diff
@@ -179,6 +179,28 @@ Once you setup your domain, depending on your SMTP provider, you will have to do
 
 If your SMTP provider uses an authentication other than `plain`, check out the [Rails docs on email configuration](https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration) for the different authentation options.
 
+## Staging server
```

Comment:
For consistency, maybe we could add a blank line between the heading and the text.